src/braket/pulse/ast/qasm_parser.py

```diff
@@ -31,13 +29,14 @@ def visit__FreeParameterExpressionIdentifier(
         self, node: ast.Identifier, context: PrinterState
     ) -> None:
         """Visit a FreeParameterExpressionIdentifier.
+
         Args:
             node (ast.Identifier): The identifier.
             context (PrinterState): The printer state context.
         """
         self.stream.write(str(node.expression.expression))
 
-    def visit_DurationLiteral(self, node: DurationLiteral, context: PrinterState) -> None:
+    def visit_DurationLiteral(self, node: ast.DurationLiteral, context: PrinterState) -> None:
```

Comment:
the linter seems to have some gauge on when should a variable be imported from a library vs when to just call library.variable

Reply:
@math411 do you know if this is documented anywhere? no worries if not

Reply:
It checks that the type hint from here matches what is in  the docs: `https://github.com/amazon-braket/amazon-braket-sdk-python/pull/778/files/591864bceda18f3bcc9953a26fa969a4031a5018#diff-41c3f660101deb414c9765b03396764961a55a03d45cbff799371ed156fec8b4R44`